src/core/seq.js

```
@@ -0,0 +1,124 @@
+    rawNums.forEach(s => {
+      const parts = s.toString().split('.');
+      if (parts[1]) maxDec = Math.max(maxDec, parts[1].length);
+    });
+  }
+  // apply formatting
+  let strs = outVals.map(v => {
+    let s;
+    if (fmt) {
+      const m = fmt.match(/^%\.([0-9]+)f$/);
+      if (m) {
+        s = v.toFixed(parseInt(m[1], 10));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)